simde/wasm/simd128.h

```diff
@@ -3579,9 +3579,9 @@ simde_wasm_i8x16_shl (simde_v128_t a, uint32_t count) {
       r_;
 
     #if defined(SIMDE_ARM_NEON_A32V7_NATIVE)
-      r_.neon_i8 = vshlq_s8(a_.neon_i8, vdupq_n_s8(HEDLEY_STATIC_CAST(int8_t, count)));
+      r_.neon_i8 = vshlq_s8(a_.neon_i8, vdupq_n_s8(HEDLEY_STATIC_CAST(int8_t, count & 7)));
```

Comment:
```suggestion
      r_.neon_i8 = vshlq_s8(a_.neon_i8, vdupq_n_s8(HEDLEY_STATIC_CAST(int8_t, count % 7)));
```
Shouldn't it be the modulo operator and not the bitwise AND operator?

Reply:
Number 2 power x are special, n % 8 == n & 7. Usually compiler will optimize code like this because bit and is much faster than div.
You can see in this example, gcc compile `argc % 8` to `argc & 7`
https://godbolt.org/z/GP69KW9YG

Reply:
Got it, thanks! Sorry for my lack of C experience, I'm the backup maintainer for SIMDe